test/jpypetest/test_imports.py

```diff
@@ -115,6 +115,20 @@ def testMRJar(self):
         self.assertTrue("B" in u)
         self.assertTrue("sub" in u)
 
+    def testAddClassPath(self):
```

Comment:
could you run a second test like this in a subprocess, in order to check the "*" path which is yet missing?

Reply:
I added a test by simply importing a module in the test directory with a second jar file as it was easier than getting the subshell stuff working.   I believe it should cover what you are looking for.   Unfortunately star is only supported at the module level so this was the cleanest solution I could come up with.

Reply:
I will likely have to disable this test for jacoco coverage tests.   Jacoco installs a "Java agent" which has a side effect of erasing the directories from jars when it recompiles the byte code.  They probably just disregarded the entries rather than passing them through.   Not sure that it is a big deal but certainly annoying for importing star.

Reply:
Actually I may have spoken too soon.  It looks like autopep8 is messing with my test.   I will have to find a way to keep the second import in place.